pkgs/by-name/ra/rattler-build/package.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  versionCheckHook,
+
+  openssl,
+  pkg-config,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rattler-build";
+  version = "0.29.0";
+
+  src = fetchFromGitHub {
+    owner = "prefix-dev";
+    repo = "rattler-build";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-MTCtDE0oX9vUIwO62sKkCGleooHU28Lq8eXzwiNPDMo=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "zip-2.2.0" = "sha256-x6hszqwalPolVn9HJMcCx7BApCOwvY4Q0NmVyFh/eEM=";
+    };
+  };
```

In the meantime `fetchCargoVendor` was merged into nixpkgs, so you should be able to remove `Cargo.lock` from the commit and just add the following:

```suggestion
  useFetchCargoVendor = true;
  cargoHash = "";
```